python-test/features/steps/agentsProvider.py

```diff
@@ -0,0 +1,55 @@
+from behave import given, when, then
+import time
+import functions
+
+base_orb_url = "https://beta.orb.live"
+email = 'tester@email.com'
+password = '12345678'
+
+agentname = 'agent1'
+agentTagKey = 'test'
+agentTagValeu = 'true'
+
+
+@given('A valid authentication')
+def get_auth(context):
+    context.token = functions.generate_token(email, password, base_orb_url)
+    assert len(context.token) > 0
+
+@when('Create an agent')
+def create_agent(context):
+    context.status_agent, context.agent_data = functions.new_agent(context.token, agentname, agentTagKey, agentTagValeu)
+    assert context.status_agent == 201
+    context.agent_key = context.agent_data['key']
+    context.agent_id = context.agent_data['id']
+    context.agent_channel_id = context.agent_data['channel_id']
+
+@when('Build agent container')
+def build_agent(context):
+    iface = 'mock'
+    provision_agent_command = f"docker run -d --net=host -e ORB_CLOUD_ADDRESS=beta.orb.live -e ORB_CLOUD_MQTT_ID={context.agent_id} -e ORB_CLOUD_MQTT_CHANNEL_ID={context.agent_channel_id} -e ORB_CLOUD_MQTT_KEY={context.agent_key} -e PKTVISOR_PCAP_IFACE_DEFAULT={iface} ns1labs/orb-agent"
```

Comment:
the way you have it, this will use the `latest` tag for `orb-agent` - this probably needs to be variable too, and often in our development pipelines it needs to be set to `develop` so that it reads: `ns1labs/orb-agent:develop`

here we would use a variable for "beta.orb.live" too